crawler/googleCrawler.py

```diff
@@ -0,0 +1,72 @@
+import googlemaps
+import json
+import requests
+import os
+import time
+
+from bs4 import BeautifulSoup
+from os.path import join, dirname
+from dotenv import load_dotenv, find_dotenv
+
+dotenv_path = join(dirname(__file__), '.env')
+load_dotenv(find_dotenv(), override=True)
+
+API_KEY = os.environ.get("API_KEY")
+gmaps = googlemaps.Client(key=API_KEY)
+
+fread = open('zipcode.txt', 'r')
+while True:
+    line = fread.readline()
+    if not line: break
+    zipcode = line[0:5]
```

Comment:
line.strip() should clean all whitespaces. I would add some test like:

zipcode = line.strip()
if not zipcode.isdigit():
  continue